doc/src/tutorials/physics/mechanics/duffing-example.rst

```diff
@@ -0,0 +1,146 @@
+
+   >>> loads = spring.to_loads() + damper.to_loads()
+
+   >>> bodies = [block_body, pendulum_body]
+
+   >>> for body in bodies:
+   ...     loads.append(me.Force(body, body.mass * g * N.y))
+
+   >>> loads
+               /      _____           3/2\                  /        _____           3/2\
+         |     /   2       /  2\   |                  |       /   2       /  2\   |
+         \k1*\/  q1   + k2*\q1 /   /*q1               \- k1*\/  q1   - k2*\q1 /   /*q1
+    [(O, ------------------------------ n_y), (block, -------------------------------- n_y), (O, c1*q1'(t) n_y), (block, -c1*q1'(t) n_y), (block, M*g n_y), (pendulum, g*m n_y)]
+                       _____                                         _____
+                      /   2                                         /   2
+                    \/  q1                                        \/  q1
+
+Lagrange's Method
+=================
+
+With the problem setup, the Lagrangian can be calculated, and the equations of motion formed.
+
+   >>> L = me.Lagrangian(N, block_body, pendulum_body)
+   >>> L
+            2      2       2     / 2       2                                     2\
+    M*q1'(t)    m*r *q2'(t)    m*\l *q2'(t)  - 2*l*sin(q2)*q1'(t)*q2'(t) + q1'(t) /
+    --------- + ------------ + ----------------------------------------------------
+        2            5                                  2
+
+   >>> LM = me.LagrangesMethod(L, [q1, q2], bodies=bodies, forcelist=loads, frame=N)
+   >>> sm.simplify(LM.form_lagranges_equations())
+    [                                       /                                    2          \   /          2\   ]
+    [-M*g + M*q1''(t) + c1*q1'(t) - g*m - m*\l*sin(q2)*q2''(t) + l*cos(q2)*q2'(t)  - q1''(t)/ + \k1 + k2*q1 /*q1]
+    [                                                                                                           ]
+    [                     /                   2                                    2        \                   ]
+    [                   m*\5*g*l*sin(q2) + 5*l *q2''(t) - 5*l*sin(q2)*q1''(t) + 2*r *q2''(t)/                   ]
+    [                   ---------------------------------------------------------------------                   ]
+    [                                                     5                                                     ]
+
+Equations of motion in [P.Brzeskia2012]_:
+
+.. math::
+
+   (M + m)y'' - ml \phi'' \sin(\phi) - ml(\phi')^2 \cos(\phi) + k_1 y + k_2 y^3 + c_1 y' = F_0 \cos(\nu t)
+
+   ml^2 \phi'' - mly'' \sin(\phi) + mlg \sin(\phi) + c_2 \phi' = 0
+
+Equations of motion in this example:
+
+.. math::
+
+   (M + m)q_1'' - mlq_2'' \sin(q_2) - ml(q_2')^2 \cos(q_2) + k_1 q_1 + k_2 q_1^3 + c_1 q_1' - (M + m)g = 0
+
+   ml^2 q_2'' - mlq_1'' \sin(q_2) + mlg \sin(q_2) + \frac{2r^2q_2''}{5} = 0
+
+The differences in the equations of motion are attributed to several factors: the gravitational force, a damping torque characterized by the damping coefficient `c_2`,
```

Comment:
I don't see $c_2$ in the equations. Is this a typo?

Reply:
$c_2$ appears in the equations of motion in the paper but not in our example because the paper introduces a damping torque, but our example does not. So it is normal that $c_2$ does not appear in line 136.

Reply:
Ah, I read the equation and then you mention c_2 so I was wondering what that is. But I hadn't read further to the next equation.

Reply:
Actually, that is wrong. I still don't see a c_2 in either of the equations you write.

Reply:
Nevermind. I'm blind.